.github/workflows/database-restore.yml

```diff
@@ -77,7 +87,35 @@ jobs:
           fi
           echo "BACKUP_FILE=${BACKUP_FILE}" >> $GITHUB_ENV
 
+      - name: Azure Login
+        if: github.event_name == 'workflow_dispatch' && inputs['skip-backup'] == true
```

Comment:
I think you might need to define the new inputs as a job env, plus it's consistent with existing setup
i.e. https://github.com/DFE-Digital/publish-teacher-training/blob/main/.github/workflows/database-restore.yml#L43 
e.g.
SKIP_BACKUP: ${{ inputs.skip-backup || 'false'  }}
SKIP_RESTORE: ${{ inputs.skip-restore || 'false'  }}